comfy/ldm/models/autoencoder.py

```diff
@@ -112,7 +114,7 @@ def __init__(
 
         self.encoder: torch.nn.Module = instantiate_from_config(encoder_config)
         self.decoder: torch.nn.Module = instantiate_from_config(decoder_config)
-        self.regularization: AbstractRegularizer = instantiate_from_config(
```

Comment:
`AbstractRegularizer` is not defined in the code. Since it's a type annotation, removing it does not alter any runtime behaviors.